detector.py

```diff
@@ -0,0 +1,168 @@
+import cv2
+import math
+import numpy as np
+from target import Target
+
+
+class Detector:
+
+    def __init__(self):
+        pass
+
+    def detectGameElement(self, array, objectsToDetect: list):
```

Comment:
Just curious, are we interested in implementing typing? I see you have the type hint for objectsToDetect.